community.html

```diff
@@ -34,7 +34,7 @@ <h2>Help others</h2>
     <li class="talk"><h3>Give a Talk</h3>
       <p>Get started with our <a href="https://github.com/sandstorm-io/sandstorm/wiki/Speaker-Kit-Lightning-Talk">speaker kit</a> to prepare your presentation. <a href="https://groups.google.com/group/sandstorm-dev">Send us an email</a> if you need help with finding a venue or preparing for your talk. We'll also send you some sandcat stickers to give out!</p>
     <li class="discuss"><h3>Discuss &amp; Answer Questions</h3>
-      <p>Development and user help happens on the <a href="https://groups.google.com/group/sandstorm-dev">mailing list</a>; join us and contribute! Real-time chat happens in IRC (<a href="https://kiwiirc.com/client/irc.libera.chat/?channel=#sandstorm">#sandstorm on libera.chat</a>), with <a href="https://libera.irclog.whitequark.org/sandstorm/">chat archives</a> available too.</p>
+      <p>Development and user help happens on the <a href="https://groups.google.com/group/sandstorm-dev">mailing list</a>; join us and contribute! Real-time chat happens on (<a href="https://matrix.to/#/#sandstorm:libera.chat">Matrix chat</a>) You can see old IRC <a href="https://libera.irclog.whitequark.org/sandstorm/">chat archives</a> available too.</p>

```

Comment:
Let's remove the parenthesis here, which only made sense when there was something outside it (IRC), and then details about it (the channel and server) inside it. I am inclined to say we can keep the chat archives for now.